Crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v4.0.0
+
+ - [changed] Firebase Crashlytics SDKs are available for general access.
```

Comment:
"The Firebase Crashlytics SDK is now available for general access."

This should be singular since this release note is specific to iOS, right?

Reply:
Yeah that makes sense 👍